clientdb/db.go

```diff
@@ -15,6 +17,13 @@ const (
 	// dbFilePermission is the default permission the client database file
 	// is created with.
 	dbFilePermission = 0600
+
+	// DefaultPoolDBTimeout is the default maximum time we wait for the
+	// Pool bbolt database to be opened. If the database is already opened
+	// by another process, the unique lock cannot be obtained. With the
+	// timeout we error out after the given time instead of just blocking
+	// for forever.
+	DefaultPoolDBTimeout = 5 * time.Second
```

Comment:
Nit: no need to export.

Reply:
I unified it with the macaroon database timeout and removed `macDatabaseOpenTimeout`, so it now needs to be exported.